pkgs/development/python-modules/buildbot/plugins.nix

```diff
@@ -6,11 +6,13 @@
     inherit (buildbot-pkg) version;
 
     src = fetchPypi {
-      inherit pname version;
-      sha256 = "0l4kyxh62y86cw97101gjs42n1sdw1n18cgh6mm337gzjn42nv3x";
+      pname = "buildbot_www";
+      inherit version format;
+      python = "py3";
+      sha256 = "1wbkkv046h6yb0fnh39ypva4x809dphjwvzv43dpgpnshlya4aws";
     };
 
-    buildInputs = [ buildbot buildbot-pkg mock ];
```

Comment:
These should have been `checkInputs`? If yes, we could use `buildbot.overrideAttrs { doCheck = false; checkInputs = []; }`.

Reply:
No, if they are not available, the build fails with:
```Please install buildbot, buildbot_pkg, and mock modules in order to install that package, or use the pre-build .whl modules available on pypi```

See: https://github.com/buildbot/buildbot/blob/master/www/base/setup.py#L24